### DISCUSSION TITLE
续费了一年，两年的现在提示账号过期还是免费版

### DISCUSSION THREAD

**Discussion by hujb2000** [getlantern/lantern-forum-cn#806] [General]
8月 13日续费了一年，当天显示是会员，但第二天8月14日显示账号过期。查询银行没有交易扣款记录，以为续费没有成功，于是8月15日又续费了两年，当天显示是会员，但第二天8月15显示账号过期。查询银行没有交易扣款记录，以为续费没有成功。但是到8月17日，在银行收支里突然又显示8月13日，8月15日的两笔交易扣款记录。

**Comment by MaxMKMax** [comment]
@hujb2000 嗯嗯好的. 我们这里帮您升级. 请您提供我支付单据和账号邮箱. 谢谢!

**Reply by hujb2000** [reply]
@Mike777Khoo 
![1](https://github.com/user-attachments/assets/8d12e4ca-abef-4acf-9b3e-01e7fa93147a)
![2](https://github.com/user-attachments/assets/af5e88dc-dc31-4cb7-8f8f-397fca87cd00)
Account Email: hujb2000@163.com

Thank you .
